src/ng/compile.js

```diff
@@ -3032,6 +3052,29 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
             removeWatchCollection.push(removeWatch);
             break;
 
+          case '<':
+            if (!hasOwnProperty.call(attrs, attrName)) {
+              if (optional) break;
+              attrs[attrName] = void 0;
+            }
+            if (optional && !attrs[attrName]) break;
+
+            parentGet = $parse(attrs[attrName]);
+
+            destination[scopeName] = parentGet(scope);
+
+            function onParentValueChange(newParentValue) {
+              destination[scopeName] = newParentValue;
+            }
+
+            if (definition.collection) {
+              removeWatch = scope.$watchCollection($parse(attrs[attrName]), onParentValueChange);
```

Comment:
I think we should parse it to init the child value. However, you can do this
```js
scope.$watchCollection(parentGet, ...)
```
to not parse twice.